src/main/resources/standardTestCases/tmp_dir.test

```diff
@@ -0,0 +1,13 @@
+name: tmp_dir
+testFormat: workflowsuccess
+backends: [Local, Jes]
```

Comment:
Take this line out. This is still making this test "centaurJes" only, as it's the only test setup with both Local+Jes.